test/unit/app/managers/test_HistoryManager.py

```diff
@@ -1,3 +1,5 @@
+import time
+
```

Comment:
```suggestion
import time
```

Reply:
Thanks, somehow I couldn't apply the suggestion, but I've included it in the rebase

Reply:
No problem!